src/style.css

```diff
@@ -1 +1,59 @@
-/* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: 14px;
+}
+
+.header {
+  display: flex;
+  padding-right: 40px;
+}
+
+.nav {
+  width: 100%;
+  display: flex;
+  justify-content: flex-end;
+  flex-direction: row;
+  align-items: center;
+  height: 60px;
+  background-color: white;
+}
+
+.nav__link {
+  line-height: 60px;
+  text-align: center;
+  text-decoration: none;
+  color: #000;
+  margin: 0 10px;
```

Comment:
On the mockup distance from edge of the screen to the text is 50px. So, your right link should not have right margin.
![image](https://user-images.githubusercontent.com/60885495/132719460-9188c1f5-b67f-481a-8eea-cb0ad8e818ec.png)
![image](https://user-images.githubusercontent.com/60885495/132719625-9ed4c6c9-0c1a-49f6-b74f-ca8a1c5ae1fc.png)